Remove bokeh templating code

bhilbert4 opened the pull request:
Remove bokeh_templating code from the repository.
This code is no longer used to create figures anywhere in the code base. It also has not been maintained for quite a while.

bhilbert4 committed:
Remove bokeh templating

pep8speaks commented:
Hello @bhilbert4, Thank you for updating !

* In the file [`jwql/website/apps/jwql/monitor_pages/monitor_bias_bokeh.py`](https://github.com/spacetelescope/jwql/blob/6f7eb33eabf8d8987898d4494d2988acaa868f3c/jwql/website/apps/jwql/monitor_pages/monitor_bias_bokeh.py):

> [Line 137:16](https://github.com/spacetelescope/jwql/blob/6f7eb33eabf8d8987898d4494d2988acaa868f3c/jwql/website/apps/jwql/monitor_pages/monitor_bias_bokeh.py#L137): [E124](https://duckduckgo.com/?q=pep8%20E124) closing bracket does not match visual indentation
> [Line 150:64](https://github.com/spacetelescope/jwql/blob/6f7eb33eabf8d8987898d4494d2988acaa868f3c/jwql/website/apps/jwql/monitor_pages/monitor_bias_bokeh.py#L150): [E502](https://duckduckgo.com/?q=pep8%20E502) the backslash is redundant between brackets
> [Line 152:17](https://github.com/spacetelescope/jwql/blob/6f7eb33eabf8d8987898d4494d2988acaa868f3c/jwql/website/apps/jwql/monitor_pages/monitor_bias_bokeh.py#L152): [E124](https://duckduckgo.com/?q=pep8%20E124) closing bracket does not match visual indentation



##### Comment last updated at 2024-10-07 16:28:42 UTC

bhilbert4 committed:
Merge branch 'develop' into remove-boeh-templating

bhilbert4 commented:
@mfixstsci this is ready for review. There was only one place in JWQL where bokeh templating was still being imported, but it wasn't actually being used. So I removed the import, and removed the bokeh_templating code.

mfixstsci reviewed (approved):
This is great @bhilbert4! We have been meaning to do this for a while.
